src/api/api.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'http://localhost:3000/api/v1',
```

Comment:
better create constants common file, where you save all strings what may be changed or reused and export it from there

```suggestion
  baseURL:  BASE_URL,
```